### PR TITLE
Include model.eval()

### PR DESCRIPTION
**Pull request by salehsargolzaee**
Setting all the layers in the model to evaluation mode before evaluation to turn off dropout layer.

### CODE REVIEW

**Review by ronny-udacity — CHANGES_REQUESTED** [review]
Thanks for the PR request. The `model.train()` needs to be added back if we're going to use `model.eval()`

**Comment by salehsargolzaee** [comment]
> 

I don't think it's necessary since we always evaluate the model after all epochs are done in this code, so the default mode is  "train mode" before that (after model initialization). However, it's safe to add it.

**Comment by salehsargolzaee** [comment]
> Thanks for the PR request. The `model.train()` needs to be added back if we're going to use `model.eval()`

I don't think `model.train()` is necessary since we always evaluate the model after all the epochs are done in this code, so the default mode is `train mode` before the evaluation steps. However, it's safe to add it.

**Review by ronny-udacity — APPROVED** [review]
